app/views/messages/new.slim

```diff
@@ -1,5 +1,5 @@
-- title t('site_title')
-
+-title t('description') 
+ 
```

Comment:
can you remove this whitespace character? if you don't already, I highly recommend turning on "show whitespace" in your text editor :)